ggshield/verticals/hmsl/output.py

```diff
@@ -23,6 +23,18 @@
     URL: "{url}"
 """
 
+TOO_MANY_SECRET_THRESHOLD = 100
+
+TOO_MANY_SECRET_TEMPLATE = """
+> Secret {number}
+Secret name: "{name}"
+Secret hash: "{hash}"
+Given the number of occurrences, your secret might be a template value
```

Comment:
nit: 
```suggestion
Given the number of occurrences, your secret might be a template value.
```

But I am actually wondering if the added line is not going to be lost in the output.

What do you think of always using TEMPLATE but with a `display_warning("Given the number of occurrences, your secret might be a template value.")` after it so that it stands out a bit more?